main.cpp

```diff
@@ -46,13 +46,17 @@
 #include "TransactionHistory.h"
 #include "model/TransactionHistoryModel.h"
 #include "model/TransactionHistorySortFilterModel.h"
-#include "daemon/DaemonManager.h"
 #include "AddressBook.h"
 #include "model/AddressBookModel.h"
 
+// IOS exclusions
+#ifndef Q_OS_IOS
```

Comment:
IMHO, I think it would be easier to maintain with a "feature" define instead of "os" define. (_wants to use these modifs for android but super lazy :)_ )